Formula/earthly.rb

```diff
@@ -0,0 +1,32 @@
+class Earthly < Formula
+  desc "Build automation tool for the container era"
+  homepage "https://github.com/vladaionescu/earthly"
+  url "https://github.com/vladaionescu/earthly/archive/v0.1.1.tar.gz"
+  sha256 "17ce130e957739da1b96245235aebb584a54241f69aea0802b97948172f5744e"
+  head "https://github.com/vladaionescu/earthly.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    system "go", "build", "-o", bin/"earthly",
+        "-ldflags", "-X main.DefaultBuildkitdImage=earthly/buildkitd:0.1.1 -X main.Version=0.1.1 -linkmode external",
```

Comment:
```suggestion
    system "go", "build", "-o", bin/"earth",
        "-ldflags", "-X main.DefaultBuildkitdImage=earthly/buildkitd:v0.1.1 -X main.Version=v0.1.1 -linkmode external",
```

Reply:
@vladaionescu so the binary is `earth` instead of `earthly`?

Reply:
Correct